infrastructure/helm/quantumserverless/charts/gateway/values.yaml

```
@@ -57,10 +57,10 @@ serviceAccount:
 
 podAnnotations: {}
 
-podSecurityContext: {}
-#  fsGroup: 1001
-#  runAsUser: 1001
-#  runAsGroup: 1001
+podSecurityContext: 
+  runAsUser: 1000
+  runAsGroup: 100
+  fsGroup: 100
```

```suggestion
  fsGroup: 1000
```

To access a file in IBM Cloud Object Storage with a non-root user, you must set the runAsUser and fsGroup values in your deployment to the **same** value.

---

Thanks Iskandar!